### PR TITLE
 torrenttools: Add version 0.6.2 

### LINKED ISSUE #2489: [Request] Add torrenttools to the main bucket
Repository: https://github.com/fbdtemme/torrenttools

Torrenttools is a cross-platform CUI tool to create, inspect and edit Bittorrent metadata files.

Torrenttools development team provides version-specific binary under https://github.com/fbdtemme/torrenttools/releases.

### PR DESCRIPTION
**Pull request by Erisa**
Resolves #2489

### CODE REVIEW

**Comment by rashil2000** [comment]
This project is quite new (Jan 2021), and has less than 500 stars (a common popularity metric). For security reasons, I'd not recommend adding it to the Main bucket yet.

See https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket

File: bucket/torrenttools.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.6.2",
+    "description": "Commandline tool for inspecting, creating and editing BitTorrent metafiles.",
+    "homepage": "https://github.com/fbdtemme/torrenttools",
+    "license": {
```
Comment (pratikpc) [inline bucket/torrenttools.json:5]: "license": "MIT" here would work in this case too

File: bucket/torrenttools.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "identifier": "MIT"
+    },
+    "url": "https://github.com/fbdtemme/torrenttools/releases/download/v0.6.2/torrenttools-0.6.2-windows-x86_64.msi",
```
Comment (pratikpc) [inline bucket/torrenttools.json:8]: For 64 bit applications, we prefer them being present in an architecture/64bit block

Check Mage for example

https://github.com/ScoopInstaller/Main/blob/master/bucket/mage.json#L6

Same for autoupdate block

**Comment by rashil2000** [comment]
@Erisa you can shift this to the Extras bucket in the meanwhile.

**Pull request closed without merging**